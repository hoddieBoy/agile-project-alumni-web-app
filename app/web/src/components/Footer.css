footer {
    justify-content: flex-end;
    width: 100%;
    text-align: center;
    background-color: #002855;
    color: #ffffff;
    padding: 20px 0;
    position: relative; 
    margin-top: 50px;
}
