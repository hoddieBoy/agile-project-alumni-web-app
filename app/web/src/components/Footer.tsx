import React from "react";
import 'components/Footer.css';

export default function Footer() {
    return (
        <footer>
            <div className="container">
                <span>Â© 2024 Alumni FIL</span>
            </div>
        </footer>
    );
}