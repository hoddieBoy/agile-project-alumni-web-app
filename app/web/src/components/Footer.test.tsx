import React from "react";
import Footer from "components/Footer";
import {render, screen} from "@testing-library/react";

describe("The Footer component", () => {
    test("renders without crashing", () => {
        const {container} = render(<Footer/>);

        expect(container).toMatchSnapshot();
    });

    test("contains the correct text", () => {
        render(<Footer/>);

        expect(screen.getByText("Â© 2024 Alumni FIL")).toBeInTheDocument();
    });
});
