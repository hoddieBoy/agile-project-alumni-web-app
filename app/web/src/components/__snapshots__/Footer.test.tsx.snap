// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Footer component renders without crashing 1`] = `
<div>
  <footer>
    <div
      class="container"
    >
      <span>
        Â© 2024 Alumni FIL
      </span>
    </div>
  </footer>
</div>
`;
